index.html

```diff
@@ -6075,7 +6075,7 @@ <h3>Executing Script</h3>
 
 <p>If at any point during the algorithm a <a>user prompt</a> appears,
  abort all subsequent substeps of this algorithm, and return
- <a>Completion</a> { [[\Type]]: <code>normal</code>, [[\Value]]: <var>null</var>, [[\Target]]: <code>empty</code> }.
```

Comment:
This doesn't look like it needed to be changed

Reply:
It's my understanding that `<var>` is intended for use with algorithm variables that store values. In this case and the one that follows, `null` refers to the null value, so `<var>` seems inappropriate. Can you say more?